src/main/resources/lang/de_de.yml

```diff
@@ -0,0 +1,67 @@
+      going-too-fast-chunkinfo: "&cElytra Geschwindigkeit ist in %neworold% %chunks% limitiert. Passe deine Geschwindigkeit an."
+      going-too-fast: "&cElytra Geschwindigkeit ist auf diesem Server limitiert. Passe deine Geschwindigkeit an."
+    nether-ceiling:
+      disabled-here: "&cElytren sind auf der Netherdecke deaktiviert."
+      you-are-flying-in: "&7Du fliegst in %neworold% &7%chunks%."
+      new: "neuen"
+      color-newchunks: "&4"
+      old: "alten"
+      color-oldchunks: "&a"
+      chunks: "Netherdecken-Chunks."
+      speed: "&7Geschwindigkeit: %speed% / %maxspeed%"
+      disabled-low-tps: "&cElytras sind auf der Netherdecke momentan deaktiviert, da die TPS unter %tps% liegt."
+      going-too-fast-low-tps: "&cElytra Geschwindigkeit ist während niedriger TPS reduziert. Passe deine Geschwindigkeit an."
+      going-too-fast-chunkinfo: "&cElytra Geschwindigkeit ist in %neworold% %chunks% limitiert. Passe deine Geschwindigkeit an."
+      going-too-fast: "&cElytra Geschwindigkeit ist auf der Netherdecke limitiert. Passe deine Geschwindigkeit an."
+    spawn:
+      disabled-here: "&cElytren sind in einem Blockradius von &6%radius% &cBlöcken um den Spawn reduziert."
+      you-are-flying-in: "&7Du fliegst in %neworold% &7%chunks%."
+      new: "neuen"
+      color-newchunks: "&4"
+      old: "alten"
+      color-oldchunks: "&a"
+      chunks: "Spawn-Chunks."
+      speed: "&7Geschwindigkeit: %speed% / %maxspeed%"
+      disabled-low-tps: "&cElytras sind am Spawn momentan deaktiviert, da die TPS unter %tps% liegt."
+      going-too-fast-chunkinfo: "&cElytra Geschwindigkeit in %neworold% %chunks% ist in einem Blockradius von %radius% Blöcken um den Spawn reduziert."
+      going-too-fast: "&cElytra Geschwindigkeit ist in einem Blockradius von %radius% Blöcken um den Spawn reduziert."
+redstone:
+  stop-spamming-levers: '&cHör auf Hebel zu spammen.'
+join-leave-messages:
+  join: "&7%player% ist dem Spiel beigetreten."
+  leave: "&7%player% hat das Spiel verlassen."
+  first-join: "&6%player% tritt dem Server als %players_num% Spieler zum ersten Mal bei."
+  enabled-connection-msgs: "&aVerbindungsnachrichten aktiviert"
+  disabled-connection-msgs: "&cVerbindungsnachrichten deaktiviert"
+chat:
+  chatco-fix-specify-player: "&cFüge einen Spielernamen hinzu."
+withers:
+  disabled-at-spawn: "&4Wither können in einem Blockradius von %radius% Blöcken um den Spawn nicht gespawnt werden."
+command-whitelist:
+  bad-command: "&4Unbekannter Befehl. Verwende /help für eine Befehlsliste."
+kicks:
+  masked-kick-message: "&6Verbindung getrennt"
+1b1t-options:
+  you-need-to-be-x-blocks-away-to-tp: "&9Du musst %blocks% Blöcke vom Spawn entfernt sein um dich zu teleportieren!"
+  nickname-reset-only: "&9Du kannst deinen Nickname nicht ändern, aber du kannst ihn mit /nick off zurücksetzen wenn du einen legacy nick hast!"
+  not-worldsize: "&7Das sind nicht die Worldstats. Verwende /stats für Worldstats."
```

Comment:
Are these google translated?

Reply:
Except for the spanish translation, all files were translated by native speakers who played the game at least once